### PR TITLE
fix bulk operations id on queue

### LINKED ISSUE #28288: [Issue] fix bulk operations id on queue
### Description (*)
After save all operations with insert directly to the database we lose all ids making it impossible to change the operations status after that.

### Preconditions (*)

RabbitMQ connection configured

### Steps to reproduce (*)
1. Install magento;
2. Run in console: `bin/magento queue:consumers:start async.operations.all`;
3. Go to API client ( **Talent API Tester** in my case);
4. Generate admin token;
5. Send **POST** request `/rest/default/async/bulk/V1/products` to create 2 products in my case (
one with correct data and another one with wrong "missing parameter **price**")
Screenshot from 2020-06-01 16-19-29

### Expected result (*)
In **magento_operation** table all operations have correct statuses: First product with the correct data has **1** status and second product with missing **price** parameter has **3** status

### Actual result (*)
In **magento_operation** table all operations have **4** status

### Propose
I propose to get all bulk operations before publish to queue.

### Contribution checklist (*)
 - [X] Pull request has a meaningful description of its purpose
 - [X] All commits are accompanied by meaningful commit messages
 - [ ] All new or changed code is covered with unit/integration tests (if applicable)
 - [ ] All automated tests passed successfully (all builds are green)

### PR DESCRIPTION
**Pull request by TadeuRodrigues**
### Description (*)
After save all operations with insert directly to the database we lose all ids making it impossible to change the operations status after that.

### Propose
I propose to get all bulk operations before publish to queue.

### Contribution checklist (*)
 - [X] Pull request has a meaningful description of its purpose
 - [X] All commits are accompanied by meaningful commit messages
 - [ ] All new or changed code is covered with unit/integration tests (if applicable)
 - [ ] All automated tests passed successfully (all builds are green)

### Resolved issues:
1. [x] resolves magento/magento2#28288: fix bulk operations id on queue

### CODE REVIEW

**Comment by nuzil** [comment]
Hi @TadeuRodrigues 
I need from you steps to reproduce the issue, otherwise will be super hard to review.
Thanks

**Comment by TadeuRodrigues** [comment]
Hi @nuzil 
We have this message on queue
{"id":null,"bulk_uuid":"0bfdc068-d62e-4253-952c-c762c8d212f4","topic_name":"async.magento.customer.api.accountmanagementinterface.createaccount.post","serialized_data":"{\"entity_id\":null,\"entity_link\":\"\",\"meta_information\":\"{\\\"customer\\\":{\\\"email\\\":\\\"mshaw@example.com\\\",\\\"firstname\\\":\\\"Melanie Shaw\\\",\\\"lastname\\\":\\\"Doe\\\"},\\\"password\\\":\\\"Strong-Password\\\"}\"}","result_serialized_data":null,"status":4,"result_message":null,"error_code":null}

we supposed to have this
{"id":64,"bulk_uuid":"057573f8-b03c-48ee-9544-7b9ada7dfee6","topic_name":"async.magento.customer.api.accountmanagementinterface.createaccount.post","serialized_data":"{\"entity_id\":null,\"entity_link\":\"\",\"meta_information\":\"{\\\"customer\\\":{\\\"email\\\":\\\"mshaw@example.com\\\",\\\"firstname\\\":\\\"Melanie Shaw\\\",\\\"lastname\\\":\\\"Doe\\\"},\\\"password\\\":\\\"Strong-Password\\\"}\"}","result_serialized_data":null,"status":4,"result_message":null,"error_code":null}

**Comment by okorshenko** [comment]
@magento create issue

File: app/code/Magento/AsynchronousOperations/Api/GetAllOperationsInterface.php [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+declare(strict_types=1);
+
+namespace Magento\AsynchronousOperations\Api;
+
+use Magento\AsynchronousOperations\Api\Data\OperationInterface;
+
+/**
+ * Interface for load all operations from a bulk
+ */
+interface GetAllOperationsInterface
```
Comment (lenaorobei) [inline app/code/Magento/AsynchronousOperations/Api/GetAllOperationsInterface.php]: There is no need to introduce new interface.

**Review by lenaorobei — CHANGES_REQUESTED** [review]
Please address static tests failures.

@nuzil since have some insights regarding this PR improvement. Could you please share it?

File: app/code/Magento/AsynchronousOperations/Api/GetAllOperationsInterface.php (hunk 1 above)
Comment (TadeuRodrigues) [reply app/code/Magento/AsynchronousOperations/Api/GetAllOperationsInterface.php]: removed

**Comment by lenaorobei** [comment]
@magento run all tests

**Comment by lenaorobei** [comment]
@nuzil could you please cover changes with tests?

**Comment by TadeuRodrigues** [comment]
@lenaorobei I'm working on this.

**Comment by slavvka** [comment]
@magento run all tests

**Comment by lenaorobei** [comment]
@magento run all tests

**Comment by slavvka** [comment]
@magento run all tests

**Comment by TadeuRodrigues** [comment]
@magento run all tests

**Comment by nuzil** [comment]
@magento run all tests

**Comment by nuzil** [comment]
@magento run Static Tests

**Comment by nuzil** [comment]
@magento run all tests

**Comment by lenaorobei** [comment]
@magento run all tests

**Comment by TadeuRodrigues** [comment]
@magento run Functional Tests B2B tests

**Comment by TadeuRodrigues** [comment]
@magento run all tests

**Comment by lenaorobei** [comment]
@magento run Database Compare

**Comment by lenaorobei** [comment]
@nuzil could you please cherry pick needed commits since we changed the base branch.

**Comment by lenaorobei** [comment]
Closing in favor of https://github.com/magento/magento2/pull/28414

**Pull request closed without merging**